Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -131,6 +131,23 @@ final class UIViewControllerExtensionsTests: XCTestCase {
 
         XCTAssertNil(childViewController.parent)
     }
+    
+    @available(iOS 13.0, *) @available(tvOS 13.0, *)
+    func testSetTabBarImage() {
+        let viewController = UIViewController()
+        let systemName = "message.fill"
+        let title = "Message"
+        let configuration = UIImage.SymbolConfiguration(scale: .large)
+        
+        viewController.setTabBarImage(systemName: "Nil",title: "Nil")
+        XCTAssertNil(viewController.tabBarItem.image)
+        
+        let testImage = UIImage(systemName: systemName, withConfiguration: configuration)
+        viewController.setTabBarImage(systemName: systemName,configuration: configuration , title: title)
+    
+        XCTAssertEqual(viewController.tabBarItem.image?.isSymbolImage, testImage?.isSymbolImage)
```

Comment:
I think you can get rid of `testImage` and be more explicit about the expected result.
```suggestion
        XCTAssertEqual(viewController.tabBarItem.image?.isSymbolImage, true)
```

Reply:
Thank you for taking the time to review my code and recommendations, all the issues were addressed.